content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
+
+To check your marker matches one of these types, you can use the `in` operator:
+
+```python
+marker = r.camera.see()[0]  # Only the closest marker
+
+marker.id in WALL
+>>> True 
+```
+
+To see the full range of constants, and the exact IDs they match up to, have a look at [the source](https://github.com/sourcebots/robot-api/blob/master/robot/game_specific.py).
+
+### _Wall_ Markers
+These markers are the ones positioned around the arena, there are 28 in total. Each token has a unique ID, and its location is documented in [the rules](/rules). This may be useful to calculate your position in the arena.
+
+```python
+marker.id in WALL  # Is this marker a wall marker
+```
+
+{{% notice tip %}}
+It's recommended to use `marker.is_wall_marker()` instead of `marker.id in WALL`
+{{% /notice %}}
+
+
+### _Column_ Markers
+These markers are those positioned on the 4 faces of the 4 columns in the arena. Each face has a unique, documented ID, and so could be used in addition to [_Wall_ Markers](#wall-markers) to calculate your position, and make sure you don't run into them!
+
+```python
+marker.id in COLUMN
+>>> True
+```
+
+Each column has its own constant to check if the marker belongs to a specific column. There's also additional constants to check if a marker is facing a certain direction, and of course checking a specific face of a specific column.
+
+{{% notice tip %}}
+Columns are referenced like the points of a compass (_N_, _E_, _S_, _W_), with the North West corner as corner 0.
+{{% /notice %}}
+
+```python
+marker.id in COLUMN  # Is the marker a column marker?
+ 
+marker.id in COLUMN_N  # Is the marker on the North column? 
+
+marker.id in COLUMN_FACING_E  # Is the marker facing East on its column?
+ 
+marker.id in COLUMN_S_FACING_W  # Is the marker on the South column facing West?
```

Comment:
Looking at the [source](https://github.com/sourcebots/robot-api/blob/master/robot/game_specific.py#L27), it appears `COLUMN_S_FACING_W` and the like are single marker IDs rather than sets, and so we need to use `==` rather than `in` here. @Adimote could you confirm this is the case?

Personally I'm not a fan of the inconsistency and would prefer to make `COLUMN_S_FACING_W` a set in `robot-api` instead, but I'm aware we've already shipped it.

Reply:
Looking at the source, youre right. I'll change to `==`. I think this is the right decision, as using `in` would imply there's more than 1 id for `COLUMN_S_FACING_W`

Reply:
Sounds good to me. In that case I think it's be a good idea to explain *why* we use `in` for some of the constants and `==` for others (i.e. some are sets of marker IDs, others are just a single marker ID).